NEWS

```diff
@@ -1,3 +1,8 @@
+* libsixel 1.10.2 (2021-10-01)
+ * new maintainer, @wsluser
```

Comment:
you'll need to get @ctrlcctrlv to agree to new maintainer

Reply:
I only put this as I wasn't sure if I needed to. Could I say contributor or do I need to say anything? I wanted to ensure that the change was documented for users when the next release happens but saw the format listing and thought it was simply a way to give credit. I'm not interested in being an actual maintainer, just contribute when I can for things I feel comfortable with.  Let me know what I should put here or just revert the commit.

Reply:
I got a different entry here I can put so I don't need to say maintainers. I just glossed over other contributor entries as they were not where I expected them mentioned at first.